docs/en/build.md

```diff
@@ -8,7 +8,7 @@ Note that the *default configuration in the C header files* (`user_config.h`, `u
 NodeMCU "application developers" just need a ready-made firmware. There's a [cloud build service](http://nodemcu-build.com/) with a nice UI and configuration options for them.
 
 ## Docker Image
-Occasional NodeMCU firmware hackers don't need full control over the complete tool chain. They might not want to setup a Linux VM with the build environment. Docker to the rescue. Give [Docker NodeMCU build](https://hub.docker.com/r/marcelstoer/nodemcu-build/) a try.
+Occasional NodeMCU firmware hackers don't need full control over the complete tool chain. They might not want to setup a Linux VM with the build environment. Docker to the rescue. Give [Docker NodeMCU build](https://hub.docker.com/r/marcelstoer/nodemcu-build/) a try.  Very useful for Windows users that need custom builds.
```

Comment:
Maybe you should include "in particular" in that sentence. Linux users can use the full toolchain as described below and Mac users have a chance to adjust the toolchain to their OS (very similar to Linux).